script/build.py

```diff
@@ -37,7 +37,8 @@ def main():
     'skia_use_system_harfbuzz=false',
     'skia_pdf_subset_harfbuzz=true',
     'skia_use_system_icu=false',
-    'skia_enable_skottie=true'
+    'skia_enable_skottie=true',
+    'skia_use_egl=true'
```

Comment:
> I am not sure if this has any side effects for other platforms. Alternatively, I could enable it only for Linux arm64.

I expect that enabling it for all platforms would just increase the binary size. 
I think it's better to enable it only for the targets where it's required.